### DISCUSSION TITLE
[Question] Will the Samsung app still work with this?

### DISCUSSION THREAD

**Discussion by elbowz** [omerfaruk-aran/esphome_samsung_hvac_bus#224] [Questions] [answered]
### Your Question

If I install the ESP32 and configure it correctly, will I still be able to control it with the standard Samsung app?

### Additional Context

Silly question, but I haven't found the answer. 

### Pre-Submission Checklist

- [X] I have searched for a similar question in open/closed issues.
- [X] I have checked the project’s documentation for relevant information.
- [X] I have provided as much detail as possible to help answer my question.

### HVAC System Type

NASA (Bus Protocol)

**Comment by brazoayeye** [comment, marked as answer]
Yes, it's a 485 network. Multiple devices can read and write data concurrently
